helix-term/src/ui/editor.rs

```diff
@@ -578,6 +578,40 @@ impl EditorView {
         }
     }
 
+    /// Render bufferline at the top
+    pub fn render_bufferline(editor: &Editor, viewport: Rect, surface: &mut Surface) {
+        let pb = PathBuf::from(SCRATCH_BUFFER_NAME); // default filename to use for scratch buffer
+        surface.clear_with(viewport, editor.theme.get("ui.statusline"));
```

Comment:
Could we have a `ui.bufferline.background` style here ?

```suggestion
        surface.clear_with(
            viewport,
            editor
                .theme
                .try_get("ui.bufferline.background")
                .unwrap_or_else(|| editor.theme.get("ui.statusline")),
        );
```